CONTRIBUTING.rst

```diff
@@ -10,7 +10,7 @@ design enhancements issues as well as pull Requests to make *coala* more
 convenient, functional and bug free.
 
 To begin contributing have a look at the open issues labeled with
-difficulty/newcomer - `newcomer issues <https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__.
+difficulty/newcomer - `newcomer issues <https://github.com/coala/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__.
```

Comment:
i think we have a nicer link for this, coala.io/new or smt? @sils1297 
or if we dont, we should i guess